front/style/layout/_loader.scss

```diff
@@ -37,3 +38,50 @@
         }
     }
 }
+
+// Loader 2
+.spin-loader,
+.spin-loader:after {
+    border-radius: 50%;
+    width: 35px;
+    height: 35px;
+}
+.spin-loader {
+    display: inline-block;
+    vertical-align: middle;
```

Comment:
Does it work only with `table-cell` display value?

Reply:
not necessarily :)